snap/local/hooks/cmd/install/install.go

```diff
@@ -323,7 +348,19 @@ func installProxy() error {
 }
 
 func installRedis() error {
-	if err := os.MkdirAll(hooks.SnapData+"/redis", 0755); err != nil {
+	setupScriptPath, err := exec.LookPath("install-setup-redis.sh")
```

Comment:
minor nitpick, but this whole file could be implemented in Go with less lines of code than running the script here:

```go
snapData := os.Getenv("SNAP_DATA")
fileName := filepath.Join(snapData,"/redis/conf/redis.conf")
if _, err := os.Stat(filepath.Join(snapData,"redis")); err != nil {
    // dir doesn't exist
    if err := os.MkdirAll(filepath.Dir(fileName),0755); err != nil {
        return err
    }
    if err := ioutil.WriteFile(fileName, nil, 0644); err != nil {
         return err
    }
}
```

with better comments of course